code/starfield/starfield.cpp

```diff
@@ -2517,27 +2517,27 @@ const char *stars_get_name_from_instance(int index, bool is_a_sun)
 }
 
 // WMC/Goober5000
-void stars_set_nebula(bool activate)
+void stars_set_nebula(bool activate, float range)
 {
     The_mission.flags.set(Mission::Mission_Flags::Fullneb, activate);
 	
     if (activate)
 	{
 		Toggle_text_alpha = TOGGLE_TEXT_NEBULA_ALPHA;
 		HUD_contrast = 1;
-		if(Fred_running) {
```

Comment:
I'm unclear on why special treatment while FRED is running is no longer needed?

Reply:
Because the function is never called from FRED.  I suppose it doesn't hurt to put this back in, though, if that were ever changed.